math/minuit2/inc/Minuit2/ABObj.h

```diff
@@ -70,7 +70,7 @@ template <> class ABObj<vec, LAVector, double> {
 
 private:
 
-  ABObj& operator=(const ABObj&) {return *this;}
```

Comment:
@lmoneta We need to double check that this is correct. The assignment operator `ABObj& operator=(const ABObj&) {return *this;}` does not make any sense?

Reply:
See the replacement `ABObj& operator=(const ABObj&) = default;` in the line below in the diff.

Reply:
As discussed with @lmoneta : Let's just delete the private assignment operator since it's not used anyway (and did nothing).